docs/source/onboarding_resources/computing/index.rst

```diff
@@ -224,33 +224,35 @@ Some Hub pages on accessing the cluster:
 - `Science and Engineering <https://hub.ihme.washington.edu/pages/viewpage.action?pageId=72807457>`_
 - `Cost Effectiveness team <https://hub.ihme.washington.edu/display/CE/Setting+up+cluster+access>`_
 
-.. _cluster_access_putty:
-
-The cluster can be accessed from most terminals 
-on your computer. Some of the most popular ones used are: PuTTY or Bitvise for 
-Windows users and Terminal or iTerm2 for Mac users. For Windows users, most researchers use PuTTY, 
-however Bitvise is also used at IHME. For Mac users, the Terminal app comes installed 
-on your computer but some perfer to use iTerm2. 
+The cluster is accessed through the Secure Shell protocol or SSH for short. 
+To access the SSH, an SSH "client" is needed. The client is an application 
+that can make SSH connections. 
 
-Terminals most commonly used to access the cluster: 
-
-Link to download `PuTTY or Bitvise <https://www.putty.org/>`_
+Both Mac and Windows include command-line SSH clients by default. This means 
+that most terminals can be used to access the cluster. As mentioned above, 
+feel free to use any terminal you are familiar with! 
 
-Link to download `iTerm2 <https://iterm2.com/>`_
+IHME Learn provides information on accessing the cluster from the command line in 
+Module 3 within the IHME Learn training `Computational Infrastructure Level 1 <https://ihme.brightspace.com/d2l/home/7028>`_.
 
-Other information from Zeb to integrate: 
+Mac users have to use the above method for cluster access. 
+For Windows users, there are some SSH clients that come with a graphical user 
+interface (e.g., you can "click" on things rather than type commands only) 
+which are more intuitive and we recommend if you are new to this type of 
+computing work. 
 
-The cluster is accessed through the SSH ("Secure Shell") protocol, so what is needed is an SSH "client" -- an application that can make SSH connections. Both Mac and Windows (at least, in recent versions of Windows 10) include command-line SSH clients by default.
+As mentioned above, for Windows users this is PuTTY or Bitvise 
 
-PuTTY and Bitvise are both SSH clients with graphical user interfaces for Windows. For Mac, it looks like some exist (e.g. Termius) but I didn't see anything super popular.
+Link to download `PuTTY or Bitvise <https://www.putty.org/>`_
```

Comment:
This is perfect as a link for these docs, but how confusing is it that Bitvise bought putty.org and put download links for both PuTTY and Bitvise there?! 🤯 😂